packages/dashboard/src/components/DashboardFrame/index.tsx

```diff
@@ -100,10 +99,9 @@ const PageTitle = styled(Grid)(({ theme }) => ({
 }))
 
 const Containment = styled(Grid)(({ theme }) => ({
-    overflow: 'auto',
     contain: 'strict',
     [theme.breakpoints.down('lg')]: {
-        minHeight: '100vh',
+        minHeight: `calc(100vh - 64px)`,
```

Comment:
suggest, add note for magic number (`64px`)

Reply:
Well，I will add it in #3061